### ISSUE TITLE
timeout while the service is up!

### ISSUE THREAD

**Issue by NoxInmortus** [PrivateBin/docker-nginx-fpm-alpine#27]
Hello,

I noticed that privatebin log is fulled by thoses :
```
s6-supervise php-fpm8: warning: can't happen: timeout while the service is up!
s6-supervise nginx: warning: can't happen: timeout while the service is up!
```

Currently using latest docker version with Swarm, this is privatebin configuration:

```yaml
  - name: privatebin
    image: privatebin/nginx-fpm-alpine
    restart_policy: any
    mode: replicated
    replicas: 1
    env:
     TZ: Europe/Paris
     PHP_TZ: Europe/Paris
    mounts:
      - source: privatebin
        target: /srv/data
        type: volume
    networks:
      - name: net-traefik
        aliases:
          - privatebin.{{ domain_name }}
    labels:
      traefik.enable: 'true'
      traefik.docker.network: net-traefik
      traefik.http.routers.privatebin-http.entrypoints: http
      traefik.http.routers.privatebin-http.middlewares: https-redirect
      traefik.http.routers.privatebin-http.rule: Host(`privatebin.{{ domain_name }}`)
      traefik.http.routers.privatebin-https.entrypoints: https
      traefik.http.routers.privatebin-https.rule: Host(`privatebin.{{ domain_name }}`)
      traefik.http.routers.privatebin-https.tls: 'true'
      traefik.http.routers.privatebin-https.tls.options: 'default'
      traefik.http.routers.privatebin-https.middlewares: secure-headers,gzip
      traefik.http.routers.privatebin-https.service: 'privatebin-https'
      traefik.http.services.privatebin-https.loadbalancer.server.port: '8080'
```

Any clue about what is going on ?

**Comment by elrido** [comment]
It sounds like you're affected by this issue of s6 on alpine 3.13:
https://gitlab.alpinelinux.org/alpine/aports/-/issues/12346#note_142720

> [...] Alpine compiles skalibs (the main library s6 depends on) with fortify-headers, and fortify-headers incorrectly forwards the definition of the ppoll() function (which skalibs uses) on 32-bit machines.

Does your environment fit this description? (being 32bit, for example armv7 or x86)

The issue got patched and is in their "edge" release. Either it will get backported to a 3.13 dot-release and our newly built images will pick it up from there or we get it once the 3.14 release is cut and we update our image to that future release.

**Comment by NoxInmortus** [comment]
Hello,

Thanks the information, yes indeed, it runs on a raspberrypi.

Closing.

**Comment by elrido** [comment]
FYI, with alpine 3.14 released, we got our images rebuilt and the `1.3.5-alpine3.14`, `1.3.5` and `latest` tags now all contain the fixed s6 for armv7.
